linkerd/app/src/env.rs

```diff
@@ -108,6 +117,13 @@ const ENV_OUTBOUND_MAX_IDLE_CONNS_PER_ENDPOINT: &str =
 pub const ENV_INBOUND_MAX_IN_FLIGHT: &str = "LINKERD2_PROXY_INBOUND_MAX_IN_FLIGHT";
 pub const ENV_OUTBOUND_MAX_IN_FLIGHT: &str = "LINKERD2_PROXY_OUTBOUND_MAX_IN_FLIGHT";
 
+/// The maximum content length (in bytes) of request bodies that will be
+/// buffered for retries.
+///
+/// If a request body's `content-length` header is over this length, the request
+/// will not be retried. If unspecified, a default value (64k) is used.
+pub const ENV_OUTBOUND_MAX_RETRY_LENGTH: &str = "LINKERD2_PROXY_OUTBOUND_MAX_RETRY_LENGTH";
```

Comment:
wdyt about calling this `..._SIZE` to be consistent

Reply:
i went back and forth on this --- started with `length` because of the content-length header, but we can change it to `size`!